substrate/frame/support/src/lib.rs

```diff
@@ -1883,11 +1883,16 @@ pub mod pallet_macros {
 	/// }
 	/// ```
 	///
-	/// Please note that this only works for signed dispatchables and requires a signed
+	/// Please note that this only works for signed dispatchables and requires a transaction
 	/// extension such as [`pallet_skip_feeless_payment::SkipCheckIfFeeless`] to wrap the
 	/// existing payment extension. Else, this is completely ignored and the dispatchable is
 	/// still charged.
 	///
+	/// Also this will not allow accountless caller to send a transaction if some transaction
+	/// extension such as [`frame_system::CheckNonce`] is used.
```

Comment:
The intra doc links should not work or? Because `frame_system` isn't a dependency?

Reply:
Indeed, I removed the link. Somehow I thought dev-dependencies were also in scope.